pyigm/clustering/tests/utils.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import print_function, absolute_import, division, unicode_literals
+
+import numpy as np
+from astropy.table import Table
+
+
+def make_rand_gal(nrand=1000):
```

Comment:
make uniform random galaxy distribution, does not account for sensitivity function.

Reply:
ok